### DISCUSSION TITLE
different capture kits for normal and tumor samples?

### DISCUSSION THREAD

**Discussion by boyangzhao** [etal/cnvkit#1123] [General]
Hi- how do we use cnvkit when we have different capture kit (hence bait intervals) used for the normal and tumor samples (in our case, these are matched samples from same patients)? It seems with the cnvkit batch, it assumes we have used the same capture kit for both samples. Thanks!

**Comment by zhaowsong** [comment]
Hello, we also encountered the same issue, with some tumors and normals being sequenced with different kits. How did you ultimately resolve this?

**Comment by etal** [comment]
If the baits don't match, then the matched normals can't be used as CNVkit references; the coverage profiles won't be representative. Instead, treat these as tumor-only scenarios, and either use a flat reference or create a pooled one from tumor samples that have relatively few or no large CNVs (and zero out bins in the reference that look suspicious, if you're comfortable with that).
